packages/web-components/src/components/pricing-table/pricing-table-header-cell.ts

```diff
@@ -23,6 +24,31 @@ class DDSPricingTableHeaderCell extends StableSelectorMixin(DDSStructuredListHea
   @property({ reflect: true })
   type: PRICING_TABLE_HEADER_CELL_TYPES = PRICING_TABLE_HEADER_CELL_TYPES.COMPLEX;
 
+  @property()
+  private _ctaType?: String;
+
+  private _handleCTASlotChange = ({ target }) => {
+    const cta = target.assignedNodes()[0];
+
+    if (cta) {
+      this._ctaType = cta.getAttribute('cta-type').toLowerCase();
+    }
+  };
+
+  renderCTASlot() {
+    const { _ctaType: ctaType } = this;
+
+    return ctaType === CTA_TYPE.VIDEO
+      ? html`
+          <dds-video-cta-container>
+            <slot name="cta" @slotchange=${this._handleCTASlotChange}></slot>
+          </dds-video-cta-container>
```

Comment:
I appreciate the rendering of the `<dds-video-cta-container>` component here so that the adopter doesn't have to define it on the markup side, however, this is a slight divergent from the way the `<dds-video-cta-container>` usage has been documented on other components where we specifically asked adopters to wrap this video player component around the CTA component (see - https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/docs/components-cta--text#using-a-video-player), when video type is available.

I'll defer to @kennylam and @ariellalgilmore on this, but we might need to adjust this and add a note in the "Docs" of the Pricing Table component for the following markup/HTML used to keep it consistent or explore a different direction.

```
<dds-video-cta-container>
    <dds-pricing-table-header-cell-cta cta-type="video" href="1_9h94wo6b" slot="cta">Call to action</dds-pricing-table-header-cell-cta>
<dds-video-cta-container>
```

Reply:
@proeung This is a good point. However unless you see a use-case for the `composite` version of the player, where authors can alter the intended presentation of this component, I don't see a problem bundling the player inside (unless you really want to maintain that level of consistency, which I also totally get).

Reply:
@kennylam I don't have much of a preference, just wanted to point out the inconsistency.

Perhaps, it's probably best to check in with @KDarch928 since he's the one that integrated the Pricing Tables in AEM and noticed this issue. Kayle, are you currently wrapping `<dds-video-cta-container>` in the component template of `<dds-pricing-table-header-cell-cta>` already, or is it handled through a different atom component where the ES import for that component is also being defined?

Reply:
@proeung 

We do have the ```<dds-video-cta-container>``` tag wrapping around the ```<dds-pricing-table-header-cell-cta>``` tag. However, the ```<dds-video-cta-container>``` wrapper will only appear around the ```<dds-pricing-table-header-cell-cta>``` tag when the "video" link type is selected. If it makes it easier to leave the video tag wrapped around the header cell cta tag no matter the link type selection, then we can update our code easily. (see AEM code snippet below) 

```
    <dds-video-cta-container data-sly-unwrap="${linkType != 'VIDEO'}">
        <dds-pricing-table-header-cell-cta data-sly-test.valid="${(wcmmode.edit && !showPlaceholder) || href || text}"
                                           class="${linkType == 'BLOG' || linkType == 'DOWNLOAD_PDF' ? 'bx__pricing-table__button--cta-type' : ''}"
                                           cta-type="${linkType.toLowerCase}"
                                           target="${target}"
                                           href="${href}"
                                           kind="tertiary"
                                           data-autoid="dds--pricing-table-header-cell-cta">
            <sly data-sly-test="${text}">
                ${text}
            </sly>
            <span slot="icon" class="${iconClass} bx--card__cta dds-ce--cta__icon" data-sly-test="${linkType == 'BLOG' || linkType == 'DOWNLOAD_PDF'}"></span>
        </dds-pricing-table-header-cell-cta>
    </dds-video-cta-container>
```

CC: @kennylam

Reply:
@proeung  
As long as all other link types work properly, then then the easy way to approach this would be to wrap the ```<dds-video-cta-container>``` around the ```<dds-pricing-table-header-cell-cta>```.  

![Screen Shot 2022-07-13 at 4 55 54 PM](https://user-images.githubusercontent.com/88403358/178843506-82c2a89f-f5b8-4540-9111-b2764e9972f5.png)